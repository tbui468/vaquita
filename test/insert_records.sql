create database sol;
open sol;

create table planets (id int key, name string, moons int, gravity float, atmosphere bool);
insert into planets (id, name, moons, gravity, atmophere) values (1, "Mars", 2, 2.23, true);
insert into planets (id, name, moons, gravity, atmospere) values (2, "Earth", 1, 82.32, false), (3, "Venus", 0, -2.32, true);
drop table planets;

close sol;
drop database sol;
exit;
