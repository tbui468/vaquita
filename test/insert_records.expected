created database sol
opened database sol
created table planets
inserted 1 record(s) into planets
inserted 2 record(s) into planets
dropped table planets
closed database sol
dropped database sol
